drivers/Makefile.dep

```diff
@@ -199,6 +199,11 @@ ifneq (,$(filter hts221,$(USEMODULE)))
   FEATURES_REQUIRED += periph_i2c
 endif
 
+ifneq (,$(filter hx711,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_gpio
```

Comment:
Doesn't this module also depend on the gpio_util module as well?

Reply:
@leandrolanzieri If periph_common/gpio_util.c results in another module, yes.